offline-install/02.a_online_models.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+registry=registry:5000
+hf_token=""
+http_proxy=""
+https_proxy=""
```

Comment:
These should be taken as (optional) arguments in **all** scripts, not hard-coded in them so that user needs to change all of them.

See e.g: https://github.com/opea-project/GenAIInfra/blob/main/kubeai/install.sh